server/weaverbird/backends/sql_translator/steps/aggregate.py

```diff
@@ -0,0 +1,119 @@
+from weaverbird.backends.sql_translator.types import (
+    SQLPipelineTranslator,
+    SQLQuery,
+    SQLQueryRetriever,
+)
+from weaverbird.pipeline.steps import AggregateStep
+
+
+def translate_aggregate(
```

Comment:
Prévoir une refacto ça fera pas mal :) juste mettre une carte en BP Tech (et peut-être prévoir une colonne VQB serait pas mal dans le Trello)

Reply:
https://trello.com/c/Dh3Q0dBH/1145-snowflake-sql-vqb-as-a-dev-i-refactor-the-aggregate-step et j'ai ajouté la colonne